### PR TITLE
Warehouse Gem fixes and improvements

### PR DESCRIPTION
**Pull request by pawelbudziszewski**
### Fixes

- Cleaned up decal (floor markings) materials. There were 2 copies of materials: one in the top level of "assets", and one in the sub-folder. Removed the top-level copy and updated all decals.
- For each decal material added `"normal.useTexture": false` to prevent `Material property: normal.textureMap` warnings. It doesn't help.
- Fixed a few incorrect materials and properties along the scene - these were causing warnings
- Turned on cached shadows for ceiling lights. This improves fps.
- Minor fixes

### Visual Improvements

- Updated Diffuse Probe Grid. Apparently, there were 2 DPGs on the scene: one disabled, and one active, but with minimal impact on the appearance. Now there is only one, with tuned parameters.
- Added 2 Reflection Probes. RPs slightly improve the overall expression, there is however one problem with placing them in the prefab: they require to be baked after instantiating the gem in the project. Otherwise, they produce warnings in the output. To prevent this, RPs are disabled by default. Added instructions on how to enable them in the README.
- Minor improvements

### Documentation

Added a README file containing:
- A list of all prefabs included in the Gem
- Tips on creating scenes using the Gem 
- Instruction on setting the camera for the best visual effect
- Instruction on turning on Reflection Probes

This PR addresses #203

### CODE REVIEW

File: Gems/WarehouseAssets/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+## Warehouse Assets Gem ##
```
Comment (hultonha) [inline Gems/WarehouseAssets/README.md:1]: nit: You don't need the trailing `###` (heading indicators), it's sufficient to just write `### Heading`

File: Gems/WarehouseAssets/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+![](docs/images/WarehouseGem.png)
+
+### General description ###
+
+This gem contains a set of models and prefabs which can be used to create a Warehouse project suitable for robotic simulations.
```
Comment (hultonha) [inline Gems/WarehouseAssets/README.md:7]: uber nit: Warehouse -> warehouse

File: Gems/WarehouseAssets/README.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+All of the contents of this gem reside in the folder `Assets`. It comprises 2 sub-folders:
+
+- `assets` containing models, textures, and material definitions
+- `Prefabs` containing prefabs that can be used to build the scene
+
+The `Prefabs` folder has the following structure:
+
+![](docs/images/PrefabsFolder.png)
+
+All the prefabs have appropriate physics parameters including collision shape.
```
Comment (hultonha) [inline Gems/WarehouseAssets/README.md:20]: uber nit: shape -> shapes

File: Gems/WarehouseAssets/README.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+- `WarehouseBox1.prefab` 
+- `WarehouseBox2.prefab`
+- `WarehouseBox3.prefab`
+- `WarehouseBox4.prefab`
+
+#### Storage Racks ####
+
+![](docs/images/StorageRacks.png)
+
+Storage racks are created from basic components, combined in 5 different permutations. The prefabs (on the image, left to right):
```
Comment (hultonha) [inline Gems/WarehouseAssets/README.md:38]: nit: on the image -> in the image

File: Gems/WarehouseAssets/README.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+- `Storage_rack2.prefab`
+- `Storage_rack3.prefab`
+- `Storage_rack4.prefab`
+- `Storage_rack5.prefab`
+
+#### Storage Rack Packs ####
+
+![](docs/images/StorageRackPacks.png)
+
+The rack packs were created by stacking 4 storage rack prefabs in different combinations. These prefabs may simplify creating larger warehouses. The prefabs (on the image, left to right):
```
Comment (hultonha) [inline Gems/WarehouseAssets/README.md:50]: uber nit (repeated): on the image -> in the image

**Review by hultonha — APPROVED** [review]
Overall looks excellent, the README is great! I only have a few minor suggestions/edits. Thanks!

**Comment by hultonha** [comment]
Thanks for making the updates @pawelbudziszewski, I've just started a build here - https://jenkins.build.o3de.org/view/Projects/job/o3de-extras/view/change-requests/job/PR-224/1/

**Comment by michalpelka** [comment]
Could we start another build?

**Comment by hultonha** [comment]
> Could we start another build?

https://jenkins.build.o3de.org/view/Projects/job/o3de-extras/view/change-requests/job/PR-224/2/ @michalpelka 👍

**Comment by hultonha** [comment]
@michalpelka @pawelbudziszewski It annoyingly looks like AR failed again on an unrelated error 😖 Could you try please updating the branch once more, and then I can start another build. Thanks!

**Review by adamdbrw — APPROVED** [review]

**Comment by pawelbudziszewski** [comment]
Rebased to stabilization/2305

**Pull request merged**